.drone.yml

```diff
@@ -71,12 +71,10 @@ pipeline:
     commands:
       - make clean
       - make generate
-      - make vet
+      - make golangci-lint
       - make lint
```

Comment:
Do we still need this `make lint` step if golangci-lint covers it? (perhaps alias `make lint` to `make golangci-lint` just for backwards compatibility as well.

Reply:
(or instead of aliasing, just add an echo saying use `make golangci-lint` instead)

Reply:
We probably won't need it, it is indeed covered by golangci.
I will add a deprication message.

Reply:
Ok, I just noticed we're not using the normal go linter, but revive which looks pretty similar to golangci, so I'd say lets keep it. But maybe rename it, to avoid confusion?

Reply:
renaming sounds good, I think if renaming is done then a note should be added to the previous name in case we have some docs pointing to make lint.

Reply:
@techknowlogick I've renamed it.